shell/platform/windows/flutter_windows_view.cc

```diff
@@ -594,6 +594,12 @@ void FlutterWindowsView::CreateRenderSurface() {
     engine_->surface_manager()->CreateSurface(GetRenderTarget(), bounds.width,
                                               bounds.height, enable_vsync);
 
+    // The EGL context cannot be current on multiple threads.
+    // Creating the render surface runs on the platform thread and
+    // makes the EGL context current. Thus, the EGL context must be
+    // released so that the raster thread can use it for rendering.
+    engine_->surface_manager()->ClearContext();
```

Comment:
This is the fix.

Previously the GL context was released in `AngleSurfaceManager::SetVSyncEnabled`, which ensured the platform thread released the GL context at startup... but it also released the GL context in the middle of rendering if the window was resized.

Now, the GL context is only released on the startup case when the surface is created.

Reply:
So this was the cause of the window resizing bug, and that should now be fixed, right?

Reply:
Yup